vermouth/processors/generate_coordinates.py

```diff
@@ -45,18 +48,17 @@ def get_anchors(molecule, indices):
     '''
     return {
         a for ix in indices for a in molecule[ix].keys()
-        if not selectors.selector_has_position(molecule.nodes[a]) is not None
+        if selectors.selector_has_position(molecule.nodes[a])
     }
 
 
 def align_z(v):
     '''
-    Generate rotation matrix aligning z-axis onto v (and vice-versa).
+    Generate rotation matrix aligning z-axis onto vector v (and vice-versa).
```

Comment:
Is it an idea to rename `v` to `vector` here? It's a bit of a nitpick from my side, admittedly